src/images/images.controller.ts

```diff
@@ -1,7 +1,81 @@
-import { Controller } from '@nestjs/common';
+import {
+  BadRequestException,
+  Controller,
+  Param,
+  ParseIntPipe,
+  Post,
+  UploadedFile,
+  UploadedFiles,
+  UseGuards,
+  UseInterceptors,
+} from '@nestjs/common';
+import { FileFieldsInterceptor } from '@nestjs/platform-express/multer/interceptors/file-fields.interceptor';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { AuthGuard } from '@nestjs/passport';
+import {
+  PostFileUploadSwagger,
+  PostThumbnailUploadSwagger,
+  User,
+} from '../common/decorators';
 import { ImagesService } from './images.service';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
 
-@Controller('images')
+const imageType = [
+  'image/png',
+  'image/jpg',
+  'image/webp',
+  'image/jpeg',
+  'image/gif',
+];
```

Comment:
```suggestion
const IMAGE_TYPES = [
  'image/png',
  'image/jpg',
  'image/webp',
  'image/jpeg',
  'image/gif',
];
```

이미지 타입들은 상수이니 대문자 스네이크 케이스로 적용해도 좋을 것 같습니다!